quora-service/src/main/java/com/upgrad/quora/service/entity/UserEntity.java

```diff
@@ -0,0 +1,173 @@
+package com.upgrad.quora.service.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "users")
+@NamedQueries(
+        {
+                @NamedQuery(name = "getUserByEmail", query = "select u from UserEntity u where u.email=:email"),
+                @NamedQuery(name = "getUserByUserName", query = "select u from UserEntity u where u.userName=:username")
+
+        }
+)
+public class UserEntity {
```

Comment:
Please add "NotNull" Annotation for all the fields which are defined as "NOT NULL" in SQL - UUID, name, email etc.. Please cross check with the SQL definition under quora-db

Reply:
completed